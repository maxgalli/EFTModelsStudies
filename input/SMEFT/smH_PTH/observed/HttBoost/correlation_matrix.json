{
    "r_smH_PTH_450_600": {
        "r_smH_PTH_450_600": 1.0,
        "r_smH_PTH_GT600": -0.3807815967574465
    },
    "r_smH_PTH_GT600": {
        "r_smH_PTH_450_600": -0.3807815967574465,
        "r_smH_PTH_GT600": 1.0
    }
}