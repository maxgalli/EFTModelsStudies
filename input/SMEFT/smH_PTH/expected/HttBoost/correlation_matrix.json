{
    "r_smH_PTH_450_600": {
        "r_smH_PTH_450_600": 1.0,
        "r_smH_PTH_GT600": -0.07905926022015837
    },
    "r_smH_PTH_GT600": {
        "r_smH_PTH_450_600": -0.07905926022015837,
        "r_smH_PTH_GT600": 1.0
    }
}